.github/scripts/main/main.sh

```diff
@@ -52,6 +52,82 @@ make install
 export PATH="$PREFIX/bin:$PATH"
 opam --version
 
+if [ "$OPAM_DOC" = "1" ]; then
+  rm -rf src_ext/
+  make -C doc html man-html pages
+
+  if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
+    . .github/scripts/common/hygiene-preamble.sh
+    diff="git diff $BASE_REF_SHA..$PR_REF_SHA"
+    set +e
+    files="`$diff --name-only --diff-filter=A | grep 'src/.*mli'`"
```

Comment:
This would have matched `src/stubs/libacl/opamlibACL.ml` - but Git can do it directly:
```suggestion
    files="$($diff --name-only --diff-filter=A -- src/**/*.mli)"
```